### ISSUE TITLE
Is it possible to use velocity command to move agent

### ISSUE THREAD

**Issue by hcv1027** [allenai/ai2thor#150]
Is it possible to use continuous velocity (linear/angular) action just like differential drive robot?

**Comment by winthos** [comment]
Inputting velocity to move the Agent is not currently supported. The agent moves on a user-defined grid (0.25m default) rather than continuous movement using velocity or force. 

If you want to get intricate control to simulate moving with velocity, you can make the grid size very small and input many `move` actions in sequence. You could even do something like change the movement magnitude over a number of `move` actions to simulate acceleration.

If you pass in the `moveMagnitude` parameter into any `move` action, you can overwrite the grid size as the movement distance.

`event = controller.step(dict(action='MoveAhead', moveMagnitude = 0.1))`

**Comment by hcv1027** [comment]
Hi @winthos ,
Thanks for your reply, I'll try it.

**Comment by hcv1027** [comment]
Hi @winthos , I follow your suggestion, and I find a case that it will not work. If the `gridSize` set at action `Initialize` is greater than the `moveMagnitude` set at action `MoveAhead`. Then the agent will not move. For example:
```python
# Agent will not move
controller.step(dict(action='Initialize', gridSize=0.25))
controller.step(dict(action='MoveAhead', moveMagnitude=0.1))

# Agent will move
controller.step(dict(action='Initialize', gridSize=0.05))
controller.step(dict(action='MoveAhead', moveMagnitude=0.1))
```

**Comment by winthos** [comment]
Yes this is intended, there is a check to make sure the Agent cannot move less than the initialized grid size. For your use case, it's best to initialize with a small gridSize that is the smallest amount of movement you would want to do.
